### PR TITLE
Updated list for Umbraco 8

### PR DESCRIPTION
**Pull request by patrickdemooij9**
Tried to go around all packages listed and see what the compatibility is and remove any that do not support Umbraco 8.

Also added Vendr and Matryoshka as I feel like they play an important role in the Umbraco ecosystem. But I can always remove them if they disagree

### CODE REVIEW

**Comment by leekelleher** [comment]
Thanks @patrickdemooij9! I'll get around to reviewing it later on. 👍

**Comment by patrickdemooij9** [comment]
Updated the PR with the changes suggested by NikRimington.
@leekelleher is it ok to also add new packages? Or should they be done through other pull requests?

File: README.md [hunk 1]
```diff
@@ -77,70 +71,49 @@ Please note * indicates that the package is commercial or may require a license
...
 
-* [RankOne - SEO Toolkit](https://our.umbraco.org/projects/backoffice-extensions/rankone-seo-toolkit/) - provides a collection of SEO tools for Umbraco that aim to optimize your content according to the latest SEO standards.
 * [SEO Checker*](https://soetemansoftware.nl/seo-checker) - find common SEO issues in your Umbraco website.
-* [SEO Metadata](https://our.umbraco.org/projects/backoffice-extensions/seo-metadata-for-umbraco/) - Property-editor for maintaining SEO metadata against your DocTypes.
 
 ## Developer tools
 
 * [CMSImport*](https://soetemansoftware.nl/cmsimport) - import content or members from any datasource into Umbraco.
-* [Diplo Trace Log Viewer](https://our.umbraco.org/projects/developer-tools/diplo-trace-log-viewer/) - view Umbraco log files directly from the Developer section in Umbraco.
 * [Optimus](https://our.umbraco.org/projects/developer-tools/optimus) - Bundling and minification of your CSS and Javascript.
```
Comment (leekelleher) [inline README.md:91]: @patrickdemooij9 I think Optimus is v7 only (currently)?

File: README.md [hunk 2]
```diff
@@ -77,70 +71,49 @@ Please note * indicates that the package is commercial or may require a license
...
 * [uSync](https://our.umbraco.org/projects/developer-tools/usync/) - Syncing tool for reading and writing the database elements to disk.
 * [Skybrud.Umbraco.Redirects](https://our.umbraco.com/packages/website-utilities/skybrud-redirects/) - Redirects manager for Umbraco.
 * [Our HealthChecks](https://our.umbraco.com/packages/backoffice-extensions/ourumbracohealthchecks/) - Adds addtional health checks to the back office.
```
Comment (leekelleher) [inline README.md:94]: @patrickdemooij9 I'm not sure what the status is with this one, the project page on Our only has the v7 version, but on NuGet it has the v8 version. 🤔

**Review by leekelleher — APPROVED** [review]
@patrickdemooij9 Overall, great job, thank you!

I've left a couple of comments about some of the packages, whether they support v8 or not. But that doesn't hold up merging this your pull request.

**Comment by leekelleher** [comment]
Merged in! 🎉 

> @leekelleher is it ok to also add new packages? Or should they be done through other pull requests?

Generally we've been having separate pull requests for each package update, but given this was an entire list review, it's been easier to do it in bulk. 👍 

@NikRimington I'll reply on PR #60. 👍

File: README.md (hunk 1 above)
Comment (patrickdemooij9) [reply README.md:91]: You're right, that one shouldn't be on the list anymore

File: README.md (hunk 2 above)
Comment (patrickdemooij9) [reply README.md:94]: The v8 version seems to be working fine for me. Just installed it on a local project and the health checks are there combined with the ones from core

**Pull request merged**